python/lsst/daf/butler/remote_butler/_remote_butler.py

```diff
@@ -439,8 +461,7 @@ def _exists_many(
         *,
         full_check: bool = True,
     ) -> dict[DatasetRef, DatasetExistence]:
-        # Docstring inherited.
-        raise NotImplementedError()
+        return {ref: self.exists(ref, full_check=full_check) for ref in refs}
```

Comment:
This is obviously pretty inefficient. (Not radically worse than the DirectButler version, though -- it also does a getDataset query per-ref.)  I anticipate modifying the file info endpoint to support batching to implement `get_many_uris`, which could also be used to get a speedup here.